sqlite-android/build.gradle

```diff
@@ -48,7 +48,7 @@ android {
 
 dependencies {
     api 'androidx.core:core:1.9.0'
-    api 'androidx.sqlite:sqlite:2.3.0'
+    api 'androidx.sqlite:sqlite:2.3.1'
```

Comment:
This is a version bump that seems to be going in accidentally, is that intended?

Reply:
@LeoColman that intended, update of AndroidX SQLite - [https://developer.android.com/jetpack/androidx/releases/sqlite#2.3.1](url)